roles/dd-nodepool/files/nodepool_server.py

```diff
@@ -55,3 +70,10 @@ def get_nodepool_status(self, status_url, timeout, service_check_tags):
                            AgentCheck.OK,
                            message='Received status from %s' % status_url,
                            tags=service_check_tags)
+
+    def get_nodepool_images(self, images_url, timeout, images_check_tags):
+        r = requests.get(images_url, timeout=timeout)
+        currentTime = int(time.time())
+
+        for image in r.json():
+            self.gauge(self.IMAGES_CHECK_NAME, currentTime - image['age'])
```

Comment:
hmm, so i'm not entirely sure what to do here, but this won't work. If you submit each metric with the same name we will end up in datadog with a bunch of values under 'nodepool.got_images' that you can't correlate to each other.  I don't think anyone here is particularly familiar with how we want this reported in datadog yet, but we at least need to identify the image name some how. So at a first guess: 

`self.gauge('nodepool.image.%s.age' % image['name'], currentTime - image['age'])`

Also without looking I'm surprised that the entry 'age' is an absolute time, rather than an age already.